packages/office-ui-fabric-react/src/components/Persona/Persona.test.tsx

```diff
@@ -1,7 +1,8 @@
 /* tslint:disable-next-line:no-unused-variable */
 import * as React from 'react';
 import { setRTL } from '../../Utilities';
-import { Persona, PersonaInitialsColor } from './index';
+import { Persona } from './Persona';
+import { PersonaInitialsColor } from './Persona.Props';
```

Comment:
Probably not important but the preexisting index import isn't an "uber index import" that you described trying to fix in your PR description?